mis_builder/__manifest__.py

```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
 # Copyright 2014-2017 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'MIS Builder',
-    'version': '10.0.3.1.1',
+    'version': '11.0.3.1.1',
```

Comment:
by convention should be "11.0.0.0" but I understand if you want to keep internal versions in line :)

Reply:
Yes, I do the effort of maintaining feature parity across 9 10 11 so this version number expresses that.